platforms/core-runtime/launcher/src/test/groovy/org/gradle/launcher/daemon/protocol/DaemonMessageSerializerTest.groovy

```diff
@@ -28,11 +28,11 @@ import org.gradle.internal.logging.events.UserInputResumeEvent
 import org.gradle.internal.serialize.PlaceholderException
 import org.gradle.internal.serialize.Serializer
 import org.gradle.internal.serialize.SerializerSpec
-import org.gradle.tooling.internal.provider.action.BuildActionSerializer
-import org.gradle.tooling.internal.provider.action.ExecuteBuildAction
 import org.gradle.launcher.daemon.diagnostics.DaemonDiagnostics
 import org.gradle.launcher.exec.BuildActionResult
 import org.gradle.launcher.exec.DefaultBuildActionParameters
+import org.gradle.tooling.internal.provider.action.BuildActionSerializer
+import org.gradle.tooling.internal.provider.action.ExecuteBuildAction
```

Comment:
💅 The file is practically unchanged;  It's fine to leave it in, but in the future I'd remove them to minimize PR footprint.